eclipse/plugins/org.switchyard.tools.ui/src/org/switchyard/tools/ui/operations/CreateSwitchYardProjectOperation.java

```diff
@@ -484,6 +486,20 @@ public void run(IProgressMonitor monitor) throws CoreException {
         }
     }
 
+    private void refreshProject(IProgressMonitor monitor) throws CoreException {
+        // update the project so we ensure a Project->Clean is done so the
+        // MANIFEST.MF is built and we don't run into trouble deploying the 
+        // project on a Fuse server.
+        if (_projectMetatData.getNewProjectHandle() != null) {
```

Comment:
_projectMetatData.getNewProjectHandle() can be extracted as local variable as it is used 3 times